### ISSUE TITLE
Can I use draco to Compress 3DTiles？And How？

### ISSUE THREAD

**Issue by lf2020** [google/draco#543]

**Comment by FrankGalligan** [comment]
Do you have a link to what "3DTiles" you are asking about?

**Comment by lf2020** [comment]
yes,this is the link(https://github.com/CesiumGS/3d-tiles),thanks for your help



At 2020-02-15 08:11:23, "FrankGalligan" <notifications@github.com> wrote:

Do you have a link to what "3DTiles" you are asking about?

**Comment by javagl** [comment]
3D Tiles _itself_ does not directly define the storage of "geometry data". It only defines a JSON structure that is used for organizing a huge amount of 3D content, so that it can be delivered and rendered efficiently. 

That being said: The "tile content" (i.e. the actual geometry data) that 3D Tiles refers to can come in different formats. And one geometry content format that is supported by 3D Tiles is glTF. And glTF can use Draco via the [`KHR_draco_mesh_compression`](https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_draco_mesh_compression/README.md) extension. This is also supported by many viewers (for example, CesiumJS, when you want to load 3D Tiles). 

So ... yes, you can somehow use Draco to compress 3D Tiles - namely by compressing each tile content, and storing it as a glTF.
